Add complex flow example written in Rust

datalogics-cgreen opened the pull request:
Add an OCR text => Extract Text sample written in Rust. This is the first Rust example.

datalogics-cgreen committed:
Add complex flow example written in Rust

datalogics-kam commented on .gitignore:
```diff
@@ -1289,3 +1289,5 @@ Node/node_modules/
 Node/package-lock.json
 PHP/**/vendor/**
 PHP/**/composer.*
+Rust/**/Cargo.lock
```
A better way to do this is

- Go to the Edit link at the top of this file
- Add Node, PHP, and Rust to the list of ignore templates.
- Replace lines 1-1286 with the newly generated boilerplate

datalogics-cgreen replied:
I've updated the template, but I had to make some additional tweaks to enable it to find the ignorable files. I think the end result is still reasonably tidy with everything categorized.

datalogics-cgreen committed:
Update .gitignore

datalogics-cgreen committed:
Be more specific in .gitignore

- Node and PHP files to ignore were not picked up by the template .gitignore. Add more precise file locations.

datalogics-kam commented on .gitignore:
```diff
@@ -10,7 +10,8 @@ bin/
 
 # Common node modules locations
-/node_modules
+Node/node_modules
```
It might be better to do this as a patch at the end, in case the whole generated block gets replaced again, but is not critical.
